utils/mongo-utility.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+var mongoose = require('mongoose');
+
+var demoSchema;
+var DemoModel;
+
+var db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function() {
+  console.log('connection established');
+
+  // create schema
+  demoSchema = mongoose.Schema({
+    Category: {
+      type: String,
+      required: true
+    },
+    CategoryID: {
+      type: String,
+      required: true
+    },
+    MetaTitle: String,
+    MetaDescription: String
+  });
+  // create model
+  DemoModel = mongoose.model('Demo', demoSchema);
+},{
+  timestamp: true
+});
+
+mongoose.connect('mongodb://localhost/test');
+
+function writeJSONToDB(jsonObject) {
+  var tableObject = new DemoModel({
+    Category: jsonObject.Category,
+    CategoryID: jsonObject['Category ID'],
+    MetaTitle: jsonObject['Meta Title'],
+    MetaDescription: jsonObject['Meta Description']
+  });
+  console.log(tableObject._id);
```

Comment:
For using logs checkout  [winston](https://www.npmjs.com/package/winston).

Logs should be highly meaning ful since it is only way to debug a production app.